src/main/markdown/release-notes.md

```diff
@@ -62,6 +63,63 @@ The GWT Release Notes
 * * *
 
 <a id="Release_Notes_Current"></a>
+## <a id="Release_Notes_2_9_0"></a> Release Notes for 2.9.0
+
+### Highlights
+- Able to compile projects with jsinterop-base 1.0.0, elemental2 1.0.0, and
+jsinterop-annotations 2.0.0. With the exception of @JsAsync and @JsEnum, this
+brings GWT2 to be compatible across these tools with J2CL.
+
+- Added support for Java language levels 9, 10, and 11.
+
+- Officially, support is dropped for running the GWT compiler or server-side
+tooling on Java 7. The GWT distribution is still compiled to run on Java 7
+for this release, but no guarantees are made about whether or not this will 
+work. Future versions will compile bytecode for Java 8+. The release was 
+tested and found to work cross platform when run with Java 8, 11, and 14.
+
+
+### Deprecations
+- Officially deprecated the original JSNI-based elemental - with elemental2
+reaching 1.0.0, it will take over for the old elemental library, which will
+not be compatible with J2CL/GWT3. To make this clear, all classes in elemental
+have been marked as deprecated - future releases of GWT may or may not include
+this library at all.
+- Removed NoSuchMethodException emulation
+
+### Bug fixes
+- Fix Arrays.binarySearch semantics for float[] and double[]
```

Comment:
Fixes Array.....

Reply:
As you noted below, I picked out bugs from CLs, so this is verbatim the subject line of that particular change. That said, I think this was correct - `Arrays.binarySearch` is the name of the class+method? Is there another class which should have this also, called `Array`, or are you suggesting the `.` should be a space (and so "Fix array binary search semantics..." instead)?

Reply:
Re-reading this now, perhaps you were asking for the change Fix->Fixes, and not Arrays->Array?

Reply:
yes. Fixes.